kuma/static/styles/includes/_mixins.scss

```diff
@@ -41,6 +41,7 @@ Provides main mixins for use within the MDN theme.
 
 @mixin set-site-font-family() {
     font-family: $site-font-family;
+    letter-spacing: -.05px;
```

Comment:
Nit: Prefer `rem` units over `px` for typography

Reply:
Should be set here.